src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -128,4 +138,18 @@ private String getDetailedRecipe(String recipeID, HttpServletResponse response)
 
     return "Exception";
   }
+
+  private void deleteComments(String recipeID) {
```

Comment:
Would the next step here be to let only the maker of a certain comment delete that comment?

Reply:
Yeah I think so. One thing we could consider is whether deleting a post should delete all the comments on it, because we can quite easily delete the Document without deleting the subcollections.

Reply:
Yeah I think we'll want to be able to delete all comments programmatically but not let any old user have access to it

Reply:
Ok, in that case I'm thinking the entire doDelete would be gated to only allow the Recipe OP to call it.